doc/userguide/for-library-authors.adoc

```diff
@@ -173,4 +177,13 @@ endif::[]
 ifndef::env-cljdoc[]
 THIS WILL BE SHOWN EVERYWHERE ELSE
 endif::[]
-----
\ No newline at end of file
+----
+
+==== Supported rich text formats
+
+Our https://github.com/cljdoc/cljdoc/blob/master/src/cljdoc/render/rich_text.clj[rich text render] supports two formats:
+
+* Markdown
+  - We use https://github.com/vsch/flexmark-java[flexmark-java] to render Markdown, which implements the latest version of the https://commonmark.org/[CommonMark] dialect.
+* AsciiDoc
+  - We use https://asciidoctor.org/[Asciidoctor] to render http://asciidoc.org/[AsciiDoc], which implements a custom dialect of it.
```

Comment:
What do you mean with custom dialect here? Are you referring to the customisations done in the rich text namespace?

Reply:
52edff5dcd1a7f285154bcf9a39fdff9e5066cda should clarify this.